openfisca_us/parameters/hhs/ccdf/asset_limit.yaml

```diff
@@ -0,0 +1,10 @@
+description: CCDF asset limit
+
+values:
+  2021-10-01: 1_000_000
```

Comment:
Per sources in https://github.com/PolicyEngine/openfisca-us/issues/325, this was not part of the 1990 act but was introduced in the 2014 act. So please set it to 0 effective 1991-01-01 and 1_000_000 effective 2015-01-01.